<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QMainWindow" name="PreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <widget class="QWidget" name="generalPage">
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="fileLaunchActionLabel">
          <property name="text">
           <string>File Launch Action</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="fileLaunchActionComboBox">
          <item>
           <property name="text">
            <string>Do nothing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Open last file</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Show open file dialog</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="debugPage">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="3" column="1">
         <widget class="QCheckBox" name="generalChannelCheckBox">
          <property name="toolTip">
           <string>General actions in the system</string>
          </property>
          <property name="text">
           <string>General</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="generalSpecificChannelCheckBox">
          <property name="toolTip">
           <string>General actions, more fine grained</string>
          </property>
          <property name="text">
           <string>General - Specific</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="xmlChannelCheckBox">
          <property name="toolTip">
           <string>XML parsing events</string>
          </property>
          <property name="text">
           <string>XML</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="hssChannelCheckBox">
          <property name="toolTip">
           <string>HSS parsing events</string>
          </property>
          <property name="text">
           <string>HSS</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="observingChannelCheckBox">
          <property name="toolTip">
           <string>Adding, removing and notifying of observers</string>
          </property>
          <property name="text">
           <string>Observing</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QCheckBox" name="eventsChannelCheckBox">
          <property name="toolTip">
           <string>Prints messages about the event handling</string>
          </property>
          <property name="text">
           <string>Events</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QCheckBox" name="eventsSpecificChannelCheckBox">
          <property name="toolTip">
           <string>Event handling, more specific</string>
          </property>
          <property name="text">
           <string>Events - Specific</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QCheckBox" name="tokenizingChannelCheckBox">
          <property name="toolTip">
           <string>Prints the characters that are read by the tokenizer</string>
          </property>
          <property name="text">
           <string>Tokenizing</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QCheckBox" name="fullFilenamesChannelCheckBox">
          <property name="toolTip">
           <string>Normally only the filename is shown, activate this for full paths</string>
          </property>
          <property name="text">
           <string>Full Filenames</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="onChannelCheckBox">
          <property name="toolTip">
           <string>Debug on/off</string>
          </property>
          <property name="text">
           <string>On</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="overviewChannelCheckBox">
          <property name="toolTip">
           <string>Basic high level actions</string>
          </property>
          <property name="text">
           <string>Overview</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="debuggingChannelsLabel">
          <property name="text">
           <string>Debugging Channels</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="generalAction"/>
   <addaction name="debugAction"/>
  </widget>
  <action name="generalAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/General.png</normaloff>:/icons/General.png</iconset>
   </property>
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="debugAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/Debug.png</normaloff>:/icons/Debug.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>fileLaunchActionComboBox</tabstop>
  <tabstop>onChannelCheckBox</tabstop>
  <tabstop>overviewChannelCheckBox</tabstop>
  <tabstop>generalChannelCheckBox</tabstop>
  <tabstop>generalSpecificChannelCheckBox</tabstop>
  <tabstop>xmlChannelCheckBox</tabstop>
  <tabstop>hssChannelCheckBox</tabstop>
  <tabstop>observingChannelCheckBox</tabstop>
  <tabstop>eventsChannelCheckBox</tabstop>
  <tabstop>eventsSpecificChannelCheckBox</tabstop>
  <tabstop>tokenizingChannelCheckBox</tabstop>
  <tabstop>fullFilenamesChannelCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileLaunchActionComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generalAction</sender>
   <signal>triggered()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>switchPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>debugAction</sender>
   <signal>triggered()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>switchPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overviewChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generalChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generalSpecificChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>-25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xmlChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>-53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hssChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>-66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>observingChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eventsChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>717</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eventsSpecificChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tokenizingChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>680</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullFilenamesChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>onChannelCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
  <slot>preferenceChanged()</slot>
  <slot>switchPage()</slot>
 </slots>
</ui>
